cfme/tests/openstack/infrastructure/test_api_port.py

```diff
@@ -10,6 +10,7 @@
 
 
 @pytest.mark.usefixtures("setup_provider_modscope")
-def test_api_port(provider, soft_assert):
-    soft_assert(provider.summary.properties.api_port.value.isdigit(),
-                "Invalid API Port")
+def test_api_port(provider):
+    port = provider.get_yaml_data()['port']
```

Comment:
I found this yaml file value useless, since in case of SSL port changes to 13000 by default. 
Better way doing it by parsing the auth URL port
provider.get_yaml_data()['auth_url'].split(':')[1].split('/')[0]